.gitignore

```diff
@@ -12,3 +12,4 @@ dist
 node_modules
 .DS_Store
 .env
+**/*.env
```

Comment:
Put all env variables in root. Because env will be injected from root of the repo.